motion-detector/detection_events.py

```diff
@@ -0,0 +1,4 @@
+import sys
+
+def motion_events():
```

Comment:
btw. function name should have verb which describe what function is doing

Reply:
feature removed